assets/scss/base/forms.scss

```diff
@@ -0,0 +1,47 @@
+.form-control {
+  background: rgba(0, 0, 0, 0.1);
+  border: 1px solid $aluminum;
+  border-radius: 5px;
+  display: block;
+  margin: auto;
+  padding: 2rem;
+  width: 35rem;
+}
+
+.button {
+  background: rgba(0, 0, 0, 0.0);
```

Comment:
Color literals like `rgba(0, 0, 0, 0)` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>`0.0` should be written without the mantissa as `0`